src/platforms/react-native/upload-debug.mdx

```diff
@@ -10,12 +10,16 @@ Debug symbols help provide you with readable stack traces, which Sentry displays
 
 With default settings, complete stack traces are available in your Java/Kotlin error, out of the box, unless you use [enableProguardInReleaseBuilds](https://reactnative.dev/docs/signed-apk-android#enabling-proguard-to-reduce-the-size-of-the-apk-optional). In this case, you need to upload the ProGuard/R8 mapping files generated by the [Android Gradle Plugin](https://developer.android.com/studio/build/shrink-code#shrink-code), so Sentry can show you proper stack traces.
 
-Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, dwarf debug files need to be uploaded for an Android app with [NDK](https://developer.android.com/ndk/) support.
-
 ## Uploading With Sentry Gradle Plugin
 
 Upload the [Android ProGuard/R8 mapping files and native symbols](/platforms/android/proguard/) by the recommended method of using our Gradle integration.
 
+Errors raised from the native layer in Android apps require certain debug information files to be uploaded. For example, DWARF debug files need to be uploaded for an Android app with [NDK](https://developer.android.com/ndk/) support.
+
+To upload native symbols for React Native Android builds, use the Sentry Gradle Plugin's [`uploadNativeSymbols` option](/platforms/react-native/manual-setup/manual-setup/#enable-sentry-agp).
+
+To enable [Source Context](/platforms/android/data-management/debug-files/source-context/) use the [`includeSourceContext`](/platforms/react-native/manual-setup/manual-setup/#enable-sentry-agp) option.
```

Comment:
@krystofwoldrich let's enable this page for RN instead.
Eg https://docs.sentry.io/platforms/flutter/data-management/debug-files/source-context/

Reply:
On it, make sense, thanks.